v20.2/upgrade-cockroach-version.md

```diff
@@ -192,7 +196,18 @@ We recommend creating scripts to perform these steps instead of performing them
 
     If you leave versioned binaries on your servers, you do not need to do anything.
 
-1. Wait at least one minute after the node has rejoined the cluster, and then repeat these steps for the next node.
+1. After the node has rejoined the cluster, ensure that the node is ready to accept a SQL connection.
+
+    Unless there are tens of thousands of ranges on the node, it's usually sufficient to wait at least one minute. To be certain that the node is ready, run the following command:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    cockroach sql -e 'select 1'
+    ~~~
+
+    The command should complete. If it hangs, the node is not yet ready.
```

Comment:
The command always complete. The text should explain "The command will automatically wait to complete until the node is ready."